communities/ucf-new-to-research/community-assessment.md

```diff
@@ -0,0 +1,45 @@
+* Research talks are frequently given by faculty, visiting faculty, and graduate students.
+
+## Accessibility
+* MSRS has supportive officers willing to help any members, but the club as a whole has low visibility in the UCF community compared to other clubs. While the officers are helpful, executive board meetings are not accessible to the rest of the club and there is no clear path to officership. There are only a few club meetings a semester, which makes club involvement difficult if a member is working or cannot attend one of the meetings. No resources exist in the club that cater to older or less traditional students; i.e., no childcare is available and discussions tend to be focused on the traditional 4-year degree.
+* Information for UCF's undergraduate research opportunities are all online, and this information is very comprehensive.
+* UCF's Honors College is restricted only to Honors students.
+
+## Interests
+* The community I defined is specific to students who conduct research in the Computer Science field, but does not need to limited to only students majoring in Computer Science. I want the group to be open to all majors adjacent to Computer Science (Mathematics, Industrial Engineering, Linguistics, etc.) and anyone interested in interdisciplinary research.
+* Community is open to other topics (interdisciplinary research).
+
+## Goals
+* Students join research communities to gain skills and experience to prepare them to start research or pursue graduate education. Plenty of students also join who are interested in their chosen research fields, but are not yet sure if they want to pursue graduate school or a job after graduation.
+* Since Computer Science research frequently involves development, skills gained also help students find internships and jobs.
+
+## Location
+* All MSRS meetings are held on campus, typically in the evenings after classes.
+* All of UCF's official workshops are held on-campus, typically in the middle of the day at times that could conflict with student schedules.
+* Public transportation in Orlando is limited, but bus transportation to UCF is more accessible than transportation to most other locations in Orlando.
+* Space is usually ample and food is usually provided.
+* Meetings are rarely recorded and meeting minutes are rarely posted for any of the aforementioned groups.
+
+## Communication
+* MSRS communicates through the Facebook group and also has a website. They could benefit from advertising the group through other clubs or through email lists. The website is also not currently up-to-date, and should be.
+* The ACM SigAI chapter has a Facebook group, gives updates at ACM meetings, and also keeps a GitHub repo of resources.
+* UCF's official undergraduate research office communicates to students via email. Clubs evangelizing for the official office would help more students make it to workshops.
+
+## Culture
+* The communities are all very welcoming to new members.
```

Comment:
In all promotional materials that the club releases, we try to make sure there is some line specifically mentioning that new members or students new to research are welcome at meetings. I'm sure there's more that can be done on that front, however. I also added more on this to the assessment.